wdl/transforms/new-base/src/main/scala/wdl/transforms/base/wdlom2wom/graph/ScatterElementToGraphNode.scala

```diff
@@ -150,10 +150,11 @@ object ScatterElementToGraphNode {
       val graphNodeSetter = new GraphNodeSetter[CallNode]
 
       val unsatisfiedInputs = subWorkflowDefinition.inputs filter { i => !a.linkablePorts.contains(i.name) }
+      def inputNodeIdentifier(input: InputDefinition) = WomIdentifier(localName = input.name, fullyQualifiedName = a.workflowName + "." + input.name)
       val newInputNodes: Map[String, ExternalGraphInputNode] = (unsatisfiedInputs collect {
-        case i: RequiredInputDefinition => i.name -> RequiredGraphInputNode(WomIdentifier(i.name), i.womType, i.name, Callable.InputDefinition.IdentityValueMapper)
-        case i: OptionalInputDefinition => i.name -> OptionalGraphInputNode(WomIdentifier(i.name), i.womType, i.name, Callable.InputDefinition.IdentityValueMapper)
-        case i: OverridableInputDefinitionWithDefault => i.name -> OptionalGraphInputNodeWithDefault(WomIdentifier(i.name), i.womType, i.default, i.name, Callable.InputDefinition.IdentityValueMapper)
+        case i: RequiredInputDefinition => i.name -> RequiredGraphInputNode(inputNodeIdentifier(i), i.womType, a.workflowName + "." + i.name, Callable.InputDefinition.IdentityValueMapper)
+        case i: OptionalInputDefinition => i.name -> OptionalGraphInputNode(inputNodeIdentifier(i), i.womType, a.workflowName + "." + i.name, Callable.InputDefinition.IdentityValueMapper)
+        case i: OverridableInputDefinitionWithDefault => i.name -> OptionalGraphInputNodeWithDefault(inputNodeIdentifier(i), i.womType, i.default, a.workflowName + "." + i.name, Callable.InputDefinition.IdentityValueMapper)
```

Comment:
Note: This was the fix to the originally reported bug. Almost everything else in the PR was tidying up of side-effects of this change.